conda-forge.yml

```diff
@@ -17,3 +17,5 @@ os_version:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
+azure:
+  store_build_artifacts: true
```

Comment:
Any downside to just keeping this here?

Reply:
If it helps, we can keep it. This does take storage space. So once we are more confident, it might be worth dropping